sdk/platform/http_client/curl/README.md

```diff
@@ -0,0 +1,99 @@
+# Azure SDK Transport Policy for Curl
+
+Azure Transport Policy Curl (`az_curl`) provides abstractions, and helpers for communicating with Azure using libcurl in the C programming language. This library follows the Azure SDK Design Guidelines for Embedded C.
+
+The library allows clients to communicate with Azure.
+
+## Getting started
+
+### Install the package
+
+TODO
+
+### Authenticate the client
+
+TODO
+
+### Get credentials
+
+TODO
+
+### Create client
+
+TODO
+
+## Key concepts
+
+TODO
+
+## Examples
+
+TODO
+
+## Troubleshooting
+
+### General
+
+TODO
+
+### Retry policy
+
+TODO
+
+## Next steps
+
+### More sample code
+
+TODO
+
+### Additional documentation
+
+TODO
+
+## Contributing
```

Comment:
Why are we repeating all this? Please remove starting from this line down unless actually relevant to curl/http_client.

Reply:
just copied from how it is on our previous repo XD
I will remove, that's not even relevant to have it like it is

Reply:
I think we need to have one Readme and one Licence on each project
And maybe the Readme is required to have a how to contribute section..  that's my guess

Reply:
> And maybe the Readme is required to have a how to contribute section.. that's my guess

I don't know about that. I think the root readme and root contribution guide should be sufficient. Project specific readme should contain docs and design details about that project.